lib/util.js

```diff
@@ -35,7 +35,7 @@ util.getUrl = function (req) {
 
     // Remove the _escaped_fragment_ query parameter
     if (parts.query && parts.query['_escaped_fragment_'] !== undefined) {
-        
+
```

Comment:
Pourquoi ? les trailing whitespaces c'est moche non ? Même si out of topic pour cette PR...